packages/client-core/src/admin/components/Setting/Email.tsx

```diff
@@ -211,14 +211,7 @@ const Email = () => {
             name="group"
```

Comment:
Does this need to have the other usages of 'group' renamed? And change the naming in admin.json?